conf/cutout_filters/default.yaml

```diff
@@ -40,24 +44,31 @@ morphological:
     # very_large: [1000, 1000000]
   
   blur_effect:  # Blur effect threshold
-    min: 
-    max: 
+    min: .35
+    max: .42
   num_components: # Number of components range
-    min:
-    max:
+    min: 0
+    max: 16
+
+# Resize all cutouts to their mean pixel w/l. 
+# if a number besides 1 is specified resize the 
```

Comment:
Looks like the only thing we need to commit is this section. Everything else looks like your personal config settings.